### ISSUE TITLE
Master has tests failing

### ISSUE THREAD

**Issue by camillescott** [dib-lab/khmer#1171]
#1145  got merged in with borked tests:

```
-------------------- >> begin captured stdout << ---------------------
running: normalize-by-median.py in: /tmp/khmertest_dk_KX6
arguments [u'normalize-by-median.py', u'-U', u'83', u'/tmp/khmertest_dk_KX6/test.fa']


|| This is the script 'normalize-by-median.py' in khmer.
|| You are running khmer version 2.0-rc1+90.ge5df6ca.dirty
|| You are also using screed version 0.9
||
|| If you use this script in a publication, please cite EACH of the following:
||
||   * MR Crusoe et al., 2014. http://dx.doi.org/10.6084/m9.figshare.979190
||   * CT Brown et al., arXiv:1203.4802 [q-bio.GN]
||
|| Please see http://khmer.readthedocs.org/en/latest/citations.html for details.


PARAMETERS:
 - kmer size =    32            (-k)
 - n tables =     4             (-N)
 - max tablesize = 1e+06        (-x)

Estimated memory usage is 4e+06 bytes (n_tables x max_tablesize)
--------

** WARNING: tablesize is default!
** You probably want to increase this with -M/--max-memory-usage!
** Please read the docs!

Traceback (most recent call last):
  File "/mnt/home/welcherc/khmer/tests/khmer_tst_utils.py", line 112, in runscript
    status = _runscript(scriptname, sandbox=sandbox)
  File "/mnt/home/welcherc/khmer/tests/khmer_tst_utils.py", line 75, in _runscript
    exec(compile(open(scriptfile).read(), scriptfile, 'exec'), ns)
  File "/mnt/home/welcherc/khmer/tests/../scripts/normalize-by-median.py", line 372, in <module>
    main()
  File "/mnt/home/welcherc/khmer/tests/../scripts/normalize-by-median.py", line 255, in main
    args = oxutils.do_sanity_checking(args, 0.1)
  File "/mnt/home/welcherc/khmer/oxli/functions.py", line 116, in do_sanity_checking
    res = estimate_optimal_with_N_and_f(args.unique_kmers,
NameError: global name 'estimate_optimal_with_N_and_f' is not defined


--------------------- >> end captured stdout << ----------------------
```

The problem being the `do_sanity_checking` function (https://github.com/dib-lab/khmer/blob/master/oxli/functions.py#L100) not being updated with the changed names from this PR.

Please fix @bocajnotnef, and folks, let's make sure we're testing on clean environments before merging.

**Comment by bocajnotnef** [comment]
Wilco.

Sorry, folks. Don't know how I missed that.

**Comment by bocajnotnef** [comment]
Closing #1171, resolved by #1172
